.github/workflows/releases.yml

```diff
@@ -37,4 +37,36 @@ jobs:
     secrets:
       DOTTYBOT-TOKEN: ${{ secrets.DOTTYBOT_WINGET_TOKEN }}
 
+  compute-digest:
+    runs-on: ubuntu-latest
+    outputs:
+      digest: ${{ steps.digest.outputs.digest }}
+    steps:
+      - name: Compute the SHA256 of scala3-${{ inputs.version }}-x86_64-pc-win32.zip in GitHub Release
+        id: digest
+        run: |
+          curl -o artifact.zip -L https://github.com/scala/scala3/releases/download/${{ inputs.version }}/scala3-${{ inputs.version }}-x86_64-pc-win32.zip
+          echo "digest=$(sha256sum artifact.zip | cut -d " " -f 1)" >> "$GITHUB_OUTPUT"
+
+  build-chocolatey:
+    uses: ./.github/workflows/build-chocolatey.yml
+    needs: compute-digest
+    with:
+      version: ${{ inputs.version }}
+      url    : 'https://github.com/scala/scala3/releases/download/${{ inputs.version }}/scala3-${{ inputs.version }}-x86_64-pc-win32.zip'
+      digest : ${{ needs.compute-digest.outputs.digest }}
+  test-chocolatey:
+    uses: ./.github/workflows/test-chocolatey.yml
+    needs: build-chocolatey
+    with:
+      version     : ${{ inputs.version }}
+      java-version: 8
+  publish-chocolatey:
+    uses: ./.github/workflows/publish-chocolatey.yml
+    needs: [ build-chocolatey, test-chocolatey ]
+    with:
+      version: ${{ inputs.version }}
+    secrets:
+      API-KEY: ${{ secrets.CHOCOLATEY_KEY }}
+
```

Comment:
There is a beneath that line, sees to be outdated now
```
# TODO: ADD RELEASE WORKFLOW TO CHOCOLATEY AND OTHER PACKAGE MANAGERS HERE
```

Reply:
Right, the chocolatey word should be dropped